styleguide.config.js

```diff
@@ -214,6 +214,15 @@ module.exports = {
         },
       ],
     },
+    {
+      name: 'Utilities',
```

Comment:
Hooks maybe?

Reply:
:thinking: Makes sense

@emersonlaurentino do you agree?

Reply:
I believe not, because not all utilities are hooks